### PR TITLE
Move AMAGAMA_SOURCE_LANGUAGES to settings

### PR DESCRIPTION
**Pull request by phlax**
and add `en-US`

fixes a problem reported by @chazotte in gitter channel

### CODE REVIEW

File: pootle/settings/60-translation.conf [hunk 1]
```diff
@@ -51,6 +51,9 @@ POOTLE_MT_BACKENDS = [
 # the trailing slash.
 AMAGAMA_URL = 'https://amagama-live.translatehouse.org/api/v1/'
 
+# List of available source languages in AMAGAMA
+AMAGAMA_SOURCE_LANGUAGES = ('en', 'en_US', 'en-US')
```
Comment (unho) [inline pootle/settings/60-translation.conf]: amaGama only supports `en` as source language. See https://amagama-live.translatehouse.org/api/v1/languages/

Comment (unho) [inline pootle/settings/60-translation.conf]: Is not a problem to specify unsupported sources languages though, just raising that default setting is probably better to reflect what is supported by the main amaGama server.

Comment (phlax) [inline pootle/settings/60-translation.conf]: it supports en-us and en_us afaict - maybe others

**Comment by unho** [comment]
Please add new setting to release notes.

Other than this and other raised nit, lgtm.

**Comment by phlax** [comment]
ive not added to docs/release notes because its not configurable as is - altho im not clear about other `en` dialects, so i think its better as a setting

**Pull request merged**